docs/docs-source/docs/modules/run-cloudflow-gcp-marketplace/pages/run-sample-application.adoc

```diff
@@ -0,0 +1,102 @@
+= Run a Sample Cloudflow Application on GCP
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+== Setting up the Cloudflow `kubectl` Plugin
+
+With Cloudflow installed on a GKE cluster through the Marketplace, you can interact with Cloudflow via its `kubectl` plugin: `kubectl cloudflow`.
+If you haven't already, download and set up the `kubectl` plugin following the https://cloudflow.io/docs/current/get-started/prepare-development-environment.html#_download_and_install_the_cloudflow_cli[instructions].
+
+Then connect your local `kubectl` to the GKE cluster on which Cloudflow is installed. You can find the command to do that from the "Clusters" view:
+
+.Connect `kubectl` to GKE cluster
+image::connect-clusters.png[Connect kubectl to GKE cluster]
+
+It should look like the following:
+[source,bash]
+----
+$ gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <gcp-project>
+----
+Where cluster-name refers to your Kubernetes cluster, as identified https://console.cloud.google.com/kubernetes/list[here], and gcp-project refers to your https://cloud.google.com/resource-manager/docs/creating-managing-projects[Google Cloud project].
+
+== Building and Deploying call-record-aggregator
+
+Next we'll deploy a sample Cloudflow application called https://github.com/lightbend/cloudflow/tree/master/examples/call-record-aggregator[call-record-aggregator] to the cluster.
+Clone the Cloudflow https://github.com/lightbend/cloudflow[Github repo] and `cd` into the `examples/call-record-aggregator` directory. Then run the following command to build the application:
+
+[source,bash]
+----
+$ sbt buildApp
+----
+This step may take several minutes to complete, as it will build and push multiple Docker images to the Docker repository configured in `target-env.sbt`.
+
+NOTE: before running `sbt buildApp`, make sure under `call-record-aggregator` directory, you have a file named `target-env.sbt` containing the Docker registry and repository that application images will be pushed to. Refer to `target-env.sbt.example` for an example.
```

Comment:
Put this information before the `sbt buildApp` command, since readers will just follow along and immediately execute commands.

Reply:
I think you need to explain the target-env.sbt file here. Since it is GCP give an example of GCR and their project repository.